backend/comment-section.go

```diff
@@ -0,0 +1,173 @@
+// Copyright Google Inc.
```

Comment:
Why do we still need this file? Can't we get away without it? Or restrict to only the comment form endpoints.